packages/sampler/package.json

```diff
@@ -8,27 +8,28 @@
     "interlink": "npm link ../core ../i18n ../spotlight ../ui ../webos",
     "lint": "enact lint --strict",
     "test": "echo \"No test suite\" && exit 0",
-    "pack": "build-storybook -c .storybook -o dist",
-    "serve": "start-storybook -p 8080 -c .storybook --ci"
+    "pack": "storybook build -c .storybook -o dist",
+    "serve": "storybook dev -p 8080 -c .storybook --ci"
   },
   "license": "Apache-2.0",
   "dependencies": {
     "@enact/core": "^4.9.0-alpha.3",
     "@enact/i18n": "^4.9.0-alpha.3",
     "@enact/spotlight": "^4.9.0-alpha.3",
-    "@enact/storybook-utils": "^5.1.4",
+    "@enact/storybook-utils": "enactjs/storybook-utils#feature/enact_v5",
     "@enact/ui": "^4.9.0-alpha.3",
     "@enact/webos": "^4.9.0-alpha.3",
-    "@storybook/addons": "^6.5.16",
-    "@storybook/builder-webpack5": "^6.5.16",
-    "@storybook/manager-webpack5": "^6.5.16",
-    "@storybook/react": "^6.5.16",
-    "@storybook/theming": "^6.5.16",
+    "@storybook/addon-docs": "^8.1.6",
```

Comment:
And here too.

Reply:
enact sampler seems to work properly without this dependency.

But on sandstone sampler I am facing errors on every story :( I will try to find a suitable solution

![image](https://github.com/enactjs/enact/assets/63335068/4515964f-1756-473f-a629-e4b3d19d3292)